localdev/scripts/airflow-start.sh

```diff
@@ -0,0 +1,20 @@
+# download the needful
```

Comment:
Has there been testing on whether this works with the armada python code?

Reply:
yes i have tested